lib/membrane/live_compositor/live_compositor.ex

```diff
@@ -767,4 +763,23 @@ defmodule Membrane.LiveCompositor do
   defp output_group_id(output_id) do
     "output_group_#{output_id}"
   end
+
+  @spec send_req(Request.t(), :inet.port_number()) :: ApiClient.request_result()
+  defp send_req(req, lc_port) do
```

Comment:
```suggestion
  @spec handle_request(Request.t(), :inet.port_number()) :: ApiClient.request_result()
  defp handle_request(req, lc_port) do
```
plus I would pass state instead of just port